code/modules/antagonists/eldritch_cult/eldritch_book.dm

```diff
@@ -6,70 +6,70 @@
+	. += "Any mortal that reads this book will gain fascination. Baptise them with your Mansus Grasp to turn them into your disciples."
+	var/datum/antagonist/heretic/cultie = user.mind.has_antag_datum(/datum/antagonist/heretic)
+	. +=  "You have earned [cultie.get_favor_left()] favor for your deeds."
+	for (var/EK in cultie.get_all_knowledge())
+		var/datum/eldritch_knowledge/known = EK
+		if (istype(known))
+			. +=  known.desc
 
 /obj/item/forbidden_book/afterattack(atom/target, mob/user, proximity_flag, click_parameters)
-	. = ..()
 	if(!proximity_flag || !IS_HERETIC(user) || in_use)
 		return
+	var/datum/antagonist/heretic/cultie = user.mind.has_antag_datum(/datum/antagonist/heretic)
+	var/mob/living/carbon/human/victim = target
 	in_use = TRUE
-	if(istype(target,/obj/effect/eldritch))
-		remove_rune(target,user)
+	if (istype(victim) && victim.stat != DEAD && cultie.get_knowledge(/datum/eldritch_knowledge/dreamgate))
+		to_chat(user,"<span class='warning'>You open the [src] and start assaulting the mind of [victim]!</span>")
+		icon_state = "book_open"
+		flick("book_opening",src)
+		if (do_after(user,10 SECONDS,victim))
+			user.whisper("That is not dead which can eternal lie...", language = /datum/language/common)
+			var/dream_text = pick ("a hooded figurine","dead bodies, as far as the eye can see","whispering","opens a third eye","grows tentacles", "the monster of a thousand hands","beautiful creatures made out of of flesh and bone","a book... written in blood and bile")
+			to_chat(victim, "<span class='warning'>... [dream_text]...</span>")
+			if (do_after(user,10 SECONDS,victim) && !QDELETED(victim) && victim.stat != DEAD && victim.IsSleeping())
+				user.whisper("And with strange aeons even death may die...", language = /datum/language/common)
+				switch (cultie.enslave(victim))
+					if (0)
+						victim.SetSleeping(0)
+						to_chat(user,"<span class='warning'>You corrupt the mind of [victim] and is now bound to do your bidding...</span>")
+					if (3)
+						to_chat(user,"<span class='warning'>You cannot enslave this mind!</span>")
+					if (2)
+						to_chat(user,"<span class='notice'>[victim] has no mind to enslave!</span>")
+					if (1)
+						to_chat(user, "<span class='notice'>You sense a weak mind, but your powers are not strong enough to take it over!</span>")
+
+		flick("book_closing",src)
+		icon_state = initial(icon_state)
 	if(istype(target,/obj/effect/reality_smash))
-		get_power_from_influence(target,user)
-	if(istype(target,/turf/open))
-		draw_rune(target,user)
+		//Gives you a charge and destroys a corresponding influence
+		var/obj/effect/reality_smash/RS = target
+		to_chat(target, "<span class='danger'>You start drawing power from influence...</span>")
+		if(cultie && do_after(user,10 SECONDS,FALSE,RS))
+			to_chat(target, "<span class='notice'>You rupture the seal between this world and the other, increasing the influence of your Gods!</span>")	//this is never explained
+			cultie.gain_favor(5)
+			qdel(RS)
```

Comment:
Would be cool if there was some kind of temporary thing, like the reality smash becoming visible to all and fading away but it's not necessary